src/parsing.jl

```diff
@@ -244,6 +244,37 @@ end
     return block
 end
 
+###
+### Buses
+###
+
+@generated function parse_row!(rec::R, bytes, pos, len, options) where {R <: Buses30}
+    block = Expr(:block)
+    n = fieldcount(R)
+    for col in 1:(n - 2)
+        T = eltype(fieldtype(R, col))
+        push!(block.args, quote
+            rec, pos, code = parse_value!(rec, $col, $T, bytes, pos, len, options)
+        end)
+    end
+    m = n - 1
+    Tm = eltype(fieldtype(R, m))
+    Tn = eltype(fieldtype(R, n))
+    push!(block.args, quote
+        # @show (rec, pos, code)
+        pos = checkdelim!(bytes, pos, len, OPTIONS_SPACE)
+        (rec, pos, code) = parse_value!(rec, $m, $Tm, bytes, pos, len, OPTIONS_SPACE)
```

Comment:
we need to set whitespace as the delim in order to not parse the comment when parsing the second-last entry (hence `OPTIONS_SPACE`) but if we parsed with comma as the delim til now then we may be at a whitespace so we use `checkdelim!` to move to the next non-whitespace char before trying to parse this second-last entry